manifests/deploy/cluster_role.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:controller:combo
```

Comment:
Typically I've seen roles, roleBindings, serviceAccounts, and the deployment share the same name, could we call each of these resources `combo-operator`?